circuit_knitting/cutting/cutting_decomposition.py

```diff
@@ -104,8 +105,29 @@ def partition_circuit_qubits(
     return circuit
 
 
+@deprecate_func(
+    since="0.3",
+    package_name="circuit-knitting-toolbox",
+    removal_timeline="no earlier than v0.4.0",
+    additional_msg=(
+        "Instead, use ``circuit_knitting.cutting.cut_gates`` "
```

Comment:
@garrison do we need back quotes here? Is this strictly a stdio printout, or is this used by sphinx?

Reply:
I only included double backquotes because that's what terra did in the opflow deprecation message.  If it weren't for that, I probably would have used single backquotes, but nothing is required.

Reply:
I actually thought I wrote this out of habit, but thank you for your honesty hah :)